File: divisibility.py
n = input("Your number >>> ")
try:
  n = int(n)
except:
  quit("Please enter an integer.")
for i in range(n + 1):
  if i == 0:
    continue
  if n % i == 0:
    print(f"{n} is divisble by {i}!")

File: divisibility.cpp
#include <iostream>

int main() {
	int n;
	std::cout << "Your number >>> ";
	std::cin >> n;
	for (int i = 1;i < n + 1;i++) {
  		if (n % i == 0) {
    			std::cout << n;
   	 		std::cout << " is divisble by ";
    			std::cout << i << std::endl;
  		}
  	}
	return 0;
}
